packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -188,23 +232,29 @@ - (void)checkCameraAuthorization {
   }
 }
 
-- (void)checkPhotoAuthorization {
+- (void)checkPhotoAuthorization:(BOOL)pickerFlag {
```

Comment:
The name `pickerFlag` is confusing. Maybe make an enum like `ImagePickerClassType` and use the enum instead.

Reply:
Done!